### PR TITLE
DOC: Point to SurfaceTopography for details on installation

### PR DESCRIPTION
**Pull request by AntoineSIMTEK**

### CODE REVIEW

**Comment by AntoineSIMTEK** [comment]
@pastewka do you know how to strat the test.yml workflow ? I added the file but nothing happens

**Review by pastewka — APPROVED** [review]

**Comment by pastewka** [comment]
I don't remember. I thought it would run automatically.

**Comment by AntoineSIMTEK** [comment]
Ok I spotted the error: [MaskedArray.max](https://numpy.org/doc/stable/reference/generated/numpy.ma.MaskedArray.max.html) does not possess that argument. 

However, I thought we wanted to get rid of masked arrays in ContactMechanics simulations.

So the problem will probably solve itself in ContactMechnics.

**Comment by pastewka** [comment]
I've fixed the max problem in NuMPI - see here: https://github.com/IMTEK-Simulation/NuMPI/pull/56

**Comment by AntoineSIMTEK** [comment]
Now there is another problem. It seems that Flake8 is crashing ...

File: requirements.txt [hunk 1]
```diff
@@ -14,4 +14,4 @@ setuptools_scm>=3.5.0
 muFFT>=0.10.0
 runtests>=0.0.28
 SurfaceTopography>=0.93.0
-NuMPI>=0.1.6
+NuMPI>=0.2.0
```
Comment (pastewka) [inline requirements.txt:17]: I think you want 0.3.0 because that fixes the `initial` problem

**Review by pastewka — CHANGES_REQUESTED** [review]

**Review by pastewka — APPROVED** [review]

**Pull request merged**